rules/windows/initial_access_suspicious_ms_exchange_process.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2021/03/04"
+maturity = "production"
+updated_date = "2021/03/04"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+This rule identifies suspicious processes being spawned by the Microsoft Exchange Server Unified Messaging (UM) service. This
+activity has been observed from the CVE-2021-26857 exploit.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Microsoft Exchange Server UM Spawning Suspcious Processes"
+references = [
+    "https://www.microsoft.com/security/blog/2021/03/02/hafnium-targeting-exchange-servers",
+    "https://www.volexity.com/blog/2021/03/02/active-exploitation-of-microsoft-exchange-zero-day-vulnerabilities",
+]
+risk_score = 47
+rule_id = "483c4daf-b0c6-49e0-adf3-0bfa93231d6b"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.category:process and event.type:start and process.parent.executable:(UMWorkerProcess.exe and not (werfault.exe or wermgr.exe)) and winlog.event_id:(1 or 4688)
```

should this also include the service `UMService.exe`?

---

nit

```suggestion
event.category:process and event.type:start and 
  process.parent.executable:(UMWorkerProcess.exe and 
    not (werfault.exe or wermgr.exe))
```

I also think we should drop the `winlog` field so it can be compatible across all 3 indexes. It isn't needed

---

If we have concerns of casing, we could switch to eql. Also, would it be worth checking original file name here as well?

---

updated to eql for case insensitivity